lib/views/item/item-detail.js

```diff
@@ -181,12 +185,60 @@ class ItemDetail extends Component {
       <View style={styles.distance}>
         <Image
           style={styles.distanceImage}
-          source={require('../../images/item-scene-room.png')}/>
+          source={{
+            uri: 'ic_place_black_24dp',
```

Comment:
do we need to indicate the dp?

Or can we just use 'ic_place_black' for devices with different dp, also does this work in ios?